src/sentry/options/defaults.py

```diff
@@ -394,7 +394,7 @@
 # e.g. [{"project_id": 2, "message_type": "error"}, {"project_id": 3, "message_type": "transaction"}]
 register("kafka.send-project-events-to-random-partitions", default=[])
 
-# Rate to project_configs_v3
+# Rate to project_configs_v3, no longer used.
```

Comment:
I'm not sure what we do with this here?  Should it be deleted?  But then how is it deleted from the database?

Reply:
i think this is the best way to go about it. it ensures the flag name is not reused for anything else, which would mean that one could read old db values by accident